coderd/searchquery/search.go

```diff
@@ -44,6 +46,28 @@ func AuditLogs(query string) (database.GetAuditLogsOffsetParams, []codersdk.Vali
 	if !filter.DateTo.IsZero() {
 		filter.DateTo = filter.DateTo.Add(23*time.Hour + 59*time.Minute + 59*time.Second)
 	}
+
+	// Convert the "organization" parameter to an organization uuid. This can require
+	// a database lookup.
+	organizationArg := parser.String(values, "", "organization")
+	if organizationArg != "" {
+		organizationID, err := uuid.Parse(organizationArg)
+		if err == nil {
+			filter.OrganizationID = organizationID
+		} else {
+			// Organization could be a name
+			organization, err := db.GetOrganizationByName(ctx, organizationArg)
+			if err != nil {
```

Comment:
ah my bad, I fixated on line 33 but missed this check 👍

Reply:
Yea, I use the same query param of `organization`. Feels more straight forward

Reply:
Might be worth adding some sort of error if you use `organization_id` to suggest using `organization`.

Note that this param is not documented or used anywhere yet.

Reply:
Only problem is this might break existing queries, idk if any folks would have a reason to use this yet?

Reply:
> Note that this param is not documented or used anywhere yet.

Ah, if it's not documented then all bets are off.

Reply:
Yea, I added it 2 days ago, and intentionally am not documenting any of the multi-org stuff yet: https://github.com/coder/coder/pull/13663